agentassist-widget-sdk/UI/agentindex.js

```diff
@@ -2204,7 +2204,7 @@ window.AgentAssist = function AgentAssist(containerId, _conversationId, _botId,
                                                 <div class="run-info-content" id="userInput-${res._id}">
                                                     <div class="title">Customer Said - </div>
                                                     <div class="agent-utt">
-                                                        <div class="title-data">${res.agentAssistDetails.entityValue}</div>
+                                                        <div class="title-data">"${res.agentAssistDetails.entityValue}"</div>
```

Comment:
for not all we should show this double quotes right?
double quotes should be there for the one which you receive from user end

Reply:
I have asked for confirmation from Madhu about these changes. He said we need to keep irrespective the customer message.